data/language/en-GB.txt

```diff
@@ -964,27 +964,27 @@ STR_1580    :
-STR_1585    :{SMALLFONT}“This on-ride photo from {STRINGID} is really good value”
-STR_1586    :{SMALLFONT}“This on-ride photo from {STRINGID} is really good value”
-STR_1587    :{SMALLFONT}“This pretzel from {STRINGID} is really good value”
-STR_1588    :{SMALLFONT}“This hot chocolate from {STRINGID} is really good value”
-STR_1589    :{SMALLFONT}“This iced tea from {STRINGID} is really good value”
-STR_1590    :{SMALLFONT}“This funnel cake from {STRINGID} is really good value”
-STR_1591    :{SMALLFONT}“These sunglasses from {STRINGID} are really good value”
-STR_1592    :{SMALLFONT}“These beef noodles from {STRINGID} are really good value”
-STR_1593    :{SMALLFONT}“These fried rice noodles from {STRINGID} are really good value”
-STR_1594    :{SMALLFONT}“This wonton soup from {STRINGID} is really good value”
-STR_1595    :{SMALLFONT}“This meatball soup from {STRINGID} is really good value”
-STR_1596    :{SMALLFONT}“This fruit juice from {STRINGID} is really good value”
-STR_1597    :{SMALLFONT}“This soybean milk from {STRINGID} is really good value”
-STR_1598    :{SMALLFONT}“This sujeonggwa from {STRINGID} is really good value”
-STR_1599    :{SMALLFONT}“This sub sandwich from {STRINGID} is really good value”
-STR_1600    :{SMALLFONT}“This cookie from {STRINGID} is really good value”
+STR_1584    :{SMALLFONT}“This on-ride photo from {STRINGID} is a really good value”
+STR_1585    :{SMALLFONT}“This on-ride photo from {STRINGID} is a really good value”
+STR_1586    :{SMALLFONT}“This on-ride photo from {STRINGID} is a really good value”
+STR_1587    :{SMALLFONT}“This pretzel from {STRINGID} is a really good value”
+STR_1588    :{SMALLFONT}“This hot chocolate from {STRINGID} is a really good value”
+STR_1589    :{SMALLFONT}“This iced tea from {STRINGID} is a really good value”
+STR_1590    :{SMALLFONT}“This funnel cake from {STRINGID} is a really good value”
+STR_1591    :{SMALLFONT}“These sunglasses from {STRINGID} are a really good value”
+STR_1592    :{SMALLFONT}“These beef noodles from {STRINGID} are a really good value”
+STR_1593    :{SMALLFONT}“These fried rice noodles from {STRINGID} are a really good value”
+STR_1594    :{SMALLFONT}“This wonton soup from {STRINGID} is a really good value”
+STR_1595    :{SMALLFONT}“This meatball soup from {STRINGID} is a really good value”
+STR_1596    :{SMALLFONT}“This fruit juice from {STRINGID} is a really good value”
+STR_1597    :{SMALLFONT}“This soybean milk from {STRINGID} is a really good value”
+STR_1598    :{SMALLFONT}“This sujeonggwa from {STRINGID} is a really good value”
+STR_1599    :{SMALLFONT}“This sub sandwich from {STRINGID} is a really good value”
+STR_1600    :{SMALLFONT}“This cookie from {STRINGID} is a really good value”
 STR_1601    :
 STR_1602    :
 STR_1603    :
-STR_1604    :{SMALLFONT}“This roast sausage from {STRINGID} are really good value”
+STR_1604    :{SMALLFONT}“This roast sausage from {STRINGID} is a really good value”
```

Comment:
While I understand why this pull request was rejected, I do think this should actually be an is rather than an are.